packages/kyt-cli/cli/commands.js

```diff
@@ -0,0 +1,19 @@
+const program = require('commander');
+const setupAction = require('./actions/setup');
+
+const loadArgsAndDo = (action) => {
+  const args = program.args.filter(item => typeof item === 'object');
+  const flags = program.args.filter(item => typeof item === 'string');
+  action(flags, args[0]);
+};
+
+
+program
```

Comment:
another potential command this might have (just spitballing) could be `ls-starter-kyts` (or something with a better name) - lists available local starter kyts

i'd also add to the docs for the options (directory defaults to `.`, `setup` doesn't necessarily need a github url since it will prompt for static and universal, `-r` is a repo address for a starter-kyt)